### PR TITLE
Map list with Custom Component

### PR DESCRIPTION
**Pull request by Odile078**
# P.R. Description
Implementation of the Mapping Through A List And Rendering (With A Custom Component) exercise.
## Completed Tasks
- [x] Add page design
- [x] Add lintItem component
## Live link
[link](https://deploy-preview-7--clever-naiad-78d51b.netlify.app/)
## Testing
Run the command below to open the landing page on a local server for testing
- npm install
- npm run dev

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,9 +1,21 @@
...
-      <h1>react introductory exercises</h1>
+      <ul>
+        {[
+          "🐶 dog",
+          "🐱 cat",
+          "🐔 chicken",
+          "🐮 cow",
+          "🐏 sheep",
+          "🐴 horse",
+        ].map((item, index) => (
```
Comment (mercihabam) [inline src/App.jsx]: The way you use these data here doesn't look pretty cool, I would suggest storing them in a constant and then accessing them through that constant, and if possible separate the icon with the animal's name.

PS. Name also the argument `item` properly. Something like `animal`.

**Review by mercihabam — CHANGES_REQUESTED** [review]
## You did a good job

I just left one comment to take into consideration to improve the look of your code.

File: src/App.jsx [hunk 2]
```diff
@@ -2,18 +2,20 @@ import "./App.css";
 import ListItem from "./components/ListItem";
 
 function App() {
+  const animals = [
```
Comment (mercihabam) [inline src/App.jsx:5]: As this `animals` is a constant, don't forget the naming rule for a constant, try to use uppercase here.

**Review by mercihabam — CHANGES_REQUESTED** [review]
**Well done applying the changes**, however, you might have ignored one small thing which I commented on.

File: src/App.jsx (hunk 2 above)
Comment (Odile078) [reply src/App.jsx:5]: I agree that some constant variables that are global have to be named using uppercase however this particular variable is not global which is why it was not written as such.

**Review by mercihabam — APPROVED** [review]

**Pull request merged**